.circleci/config.yml

```diff
@@ -1160,6 +1655,236 @@ workflows:
+          variant: win64
+          name: build-enterprise-backend-win64
+          requires:
+            - install-grabpl
+      - build-enterprise-backend:
+          filters: *filter-only-master
+          variant: linux-x64
+          name: build-enterprise-backend-linux-x64
+          requires:
+            - install-grabpl
+      - build-enterprise-backend:
+          filters: *filter-only-master
+          variant: linux-x64-musl
+          name: build-enterprise-backend-linux-x64-musl
+          requires:
+            - install-grabpl
+      - build-enterprise-frontend:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+      - codespell:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+      - lint-go:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+      - shellcheck-grabpl:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+      - test-backend:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+            - lint-go
+      - test-frontend:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+      - mysql-integration-test:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+      - postgres-integration-test:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+      - package-oss:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+            - build-oss-backend-armv6
+            - build-oss-backend-armv7
+            - build-oss-backend-armv7-musl
+            - build-oss-backend-arm64
+            - build-oss-backend-arm64-musl
+            - build-oss-backend-osx64
+            - build-oss-backend-win64
+            - build-oss-backend-linux-x64
+            - build-oss-backend-linux-x64-musl
+            - build-oss-frontend
+            - test-backend
+            - test-frontend
+            - codespell
+            - shellcheck-grabpl
+            - mysql-integration-test
+            - postgres-integration-test
+      - package-enterprise:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+            - build-enterprise-backend-armv6
+            - build-enterprise-backend-armv7
+            - build-enterprise-backend-armv7-musl
+            - build-enterprise-backend-arm64
+            - build-enterprise-backend-arm64-musl
+            - build-enterprise-backend-osx64
+            - build-enterprise-backend-win64
+            - build-enterprise-backend-linux-x64
+            - build-enterprise-backend-linux-x64-musl
+            - build-enterprise-frontend
+            - test-backend
+            - test-frontend
+            - codespell
+            - shellcheck-grabpl
+            - mysql-integration-test
```

Comment:
wish there was ways we can define and reuse these lists (it's going to be duplicated for the production release workflow)

Reply:
@torkelo I think we might be able to re-use the same workflow for release, through branching constructs. I haven't tried it yet, but my impression is that how CircleCI intends for workflows to be used (i.e. one workflow -> many pipelines).

Reply:
Yeah, I think that makes sense.